articles/connections/passwordless/sms-gateway.md

```diff
@@ -72,32 +80,48 @@ This will return your SMS connection with your Twilio settings:
         "time_step": 300,
         "length": 6
       }
-    }
+    },
+    "is_domain_connection": false,
+    "enabled_clients": []
 }
 ```
 
-3. Send the updated configuration to the Management API using the [PATCH connections endpoint](/api/v2#!/Connections/patch_connections_by_id).
+    If your SMS Gateway accepts authenticated requests that are token-based, you can add the following to your `options` object:
 
-After updating the connection for any user that signs up or authenticates using the <dfn data-key="passwordless">Passwordless</dfn> SMS connection, the following payload will be sent to your SMS gateway:
+```json
 
+"gateway_authentication": {
+    "method": "bearer",
+    "subject": "urn:Auth0",
+    "audience": "urn:MySmsGateway",
+    "secret": "MySecretToSignTheToken"
+}
 ```
+
+    With this configuration, a JWT will be added to the `Authorization` header which contains a token with the `subject` and `audience` configured in the connection and signed with the `secret` when the payload is sent to your SMS Gateway.
```

Comment:
If you're a new user, this is confusing. What subject and audience configured in the connection? Could get away with simply referencing and linking to the how-to where this is configured.

Also, which secret? Be sure they understand which secret we're talking about. App secret? Password for their authenticated gateway that does not reside in Auth0? If I'm a newbie, I might be confused.